### PR TITLE
DERP-816 - Refactored entity bundle config.

### PR DESCRIPTION
**Pull request by lukewhitmore**
Issue [here](https://london.atlassian.net/browse/DERP-816)

N/B more PRs will be released for other bundle types.

### CODE REVIEW

**Comment by lukewhitmore** [comment]
@philwolstenholme This still has a breaking dependency on default content. What's the best way of addressing this?

File: config/install/core.entity_form_display.media.image.default.yml [hunk 1]
```diff
@@ -1,65 +1,178 @@
+uuid: eb7eb730-29b5-46de-8dd6-100cc2273046
```
Comment (Fullstack-Lincoln) [inline config/install/core.entity_form_display.media.image.default.yml:1]: Can you remove the uuid please?

File: config/install/core.entity_form_display.media.image.default.yml [hunk 2]
```diff
@@ -1,65 +1,178 @@
...
+      format_settings:
+        show_empty_fields: false
+        id: ''
+        classes: ''
+        description: ''
+        formatter: closed
+        required_fields: true
+      label: Content
+_core:
+  default_config_hash: iwj0z6LJBeD5LLt0KrCsoIPmUbRlrEcfNgxjDgN_Dlc
```
Comment (Fullstack-Lincoln) [inline config/install/core.entity_form_display.media.image.default.yml:91]: Can you remove these lines please?

File: config/install/media.type.image.yml [hunk 3]
```diff
@@ -4,10 +4,13 @@ dependencies: {  }
 id: image
 label: Image
 description: 'Use local images for reusable media.'
-source: image
+source: bynder
 queue_thumbnail_downloads: false
 new_revision: true
 source_configuration:
-  source_field: field_media_image
+  source_field: field_bynder_id
```
Comment (Fullstack-Lincoln) [inline config/install/media.type.image.yml:11]: No issue with this line but I'm just wondering as a result of this change might we need to change other components that use images and this source field?

**Review by Fullstack-Lincoln — APPROVED** [review]

File: config/install/core.entity_form_display.media.image.default.yml (hunk 1 above)
Comment (Fullstack-Lincoln) [reply config/install/core.entity_form_display.media.image.default.yml:1]: This

**Review by Fullstack-Lincoln — CHANGES_REQUESTED** [review]

**Comment by lukewhitmore** [comment]
Closed this PR it contained an API secret which has been revoked.

**Pull request closed without merging**